src/main/java/com/hillel/dzhurynskyi/homework/hm12/ToList.java

```diff
@@ -0,0 +1,13 @@
+package com.hillel.dzhurynskyi.homework.hm12;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ToList {
+    public <T> List<T> toList(T[] array) {
+        List<T> list = new ArrayList<>();
```

Comment:
try to avoid temporary variables  if you can directly return value from the method